manifests/rhel.pp

```diff
@@ -7,11 +7,47 @@
 # Opsmatic Inc. (support@opsmatic.com)
 #
 class opsmatic::rhel {
-  yumrepo { 'opsmatic_rhel_repo':
-    baseurl  => 'https://packagecloud.io/opsmatic/public/el/6/$basearch',
-    descr    => 'Opsmatic RHEL repository',
-    enabled  => '1',
-    gpgcheck => '1',
-    gpgkey   => 'file://templates/D59097AB.key';
```

Comment:
WE can't actually do this, because the repo's metadata for PackageCloud is still signed using their key. Now, their key is also available via a key server, so maybe that's how it's all still working.. but we definitely need to somehow reference both keys (I'm guessing that we need to specify the PC key for the repo itself, and then just have the Opsmatic key in the keychain for verifying individual packages, but.. I'm not 100% sure).